src/main/java/ru/itis/controllers/ProfileController.java

```diff
@@ -30,29 +30,29 @@ public ProfileController(WishListService wishListService, UserService userServic
     @GetMapping("/profile")
     @PreAuthorize("isAuthenticated()")
     @ApiOperation("View user profile page")
-    public UserDto getProfilePage(@RequestHeader(name = "AUTH") String token) {
+    public ResponseUserDto getProfilePage(@RequestHeader(name = "Authorization") String token) {
         Optional<User> userCandidate = userService.findUserByToken(token);
         if (userCandidate.isPresent()) {
```

Comment:
You can replace this long if with single `ResponseUserDto.from(defaultUserDto.orElse(defaultUserDto)`

Reply:
Plz read about Optional in Java.

Reply:
fixed